lib/modules/datasource/docker/readme.md

```diff
@@ -2,4 +2,4 @@ This datasource identifies an image's source repository according to the [pre-de
 
 This datasource looks for the metadata of the **latest stable** image found on the Docker registry and uses the value of the label `org.opencontainers.image.source` and `org.label-schema.vcs-url` as the `sourceUrl`.
 
-The [Label Schema](https://label-schema.org/) is superseded by OCI annotations, therefore `org.opencontainers.image.source` label should be preferred.
+The [Label Schema](https://label-schema.org/) is superseded by OCI annotations, so the `org.opencontainers.image.source` label should be preferred.
```

Comment:
I'm not happy with "should be preferred"  as it doesn't tell me who or what prefers something in this sentence.

Do we recommend that our readers use OCI annotations and the `org.opencontainers.image.source` label?

Reply:
"use the .... if possible" ?

Reply:
```suggestion
The [Label Schema](https://label-schema.org/) is superseded by OCI annotations, use the `org.opencontainers.image.source` label if possible.
```